index.html

```diff
@@ -401,7 +401,7 @@
     </div>
 
     <script src="lib/js/head.min.js"></script>
-    <script src="palantirnet-theme/assets/js/reveal.js"></script>
+    <script src="js/reveal.js"></script>
 
```

Comment:
So I changed the script link because I moved the script into the palantirnet-theme directory and made a few basic changes to the file in there. I wanted to package everything into a neat directory so you could just pull that along w/ you and make edits as you need.

If we want to pull down a stock reveal.js script every time (which is fine), we just need to update the readme that talks about editing the reveal.js file so that it talks about it in the correct location.